test/auth_test.go

```diff
@@ -13,8 +13,8 @@ import (
 	"github.com/nats-io/gnatsd/server"
 )
 
-func doAuthConnect(t tLogger, c net.Conn, token, user, pass string) {
-	cs := fmt.Sprintf("CONNECT {\"verbose\":true,\"auth_token\":\"%s\",\"user\":\"%s\",\"pass\":\"%s\"}\r\n", token, user, pass)
+func doAuthConnect(t tLogger, c net.Conn, token, user, pass string, trace bool) {
```

Comment:
Based on the number of changes below, wonder if leaving `doAuthConnect()` alone and add a special one that adds trace. Thoughts?